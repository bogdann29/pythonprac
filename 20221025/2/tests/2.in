print(*list(slide(range(5,15), 10)))
