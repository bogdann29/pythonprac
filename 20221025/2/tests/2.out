5 6 7 8 9 10 11 12 13 14 6 7 8 9 10 11 12 13 14 7 8 9 10 11 12 13 14 8 9 10 11 12 13 14 9 10 11 12 13 14 10 11 12 13 14 11 12 13 14 12 13 14 13 14 14
